libs/model/src/community/CreateGroup.command.ts

```diff
@@ -55,24 +56,20 @@ export function CreateGroup(): Command<typeof schemas.CreateGroup> {
             { transaction },
           );
           if (topicsToAssociate.length > 0) {
-            // add group to all specified topics
-            await models.Topic.update(
-              {
-                group_ids: sequelize.fn(
-                  'array_append',
-                  sequelize.col('group_ids'),
-                  group.id,
-                ),
-              },
-              {
-                where: {
-                  id: {
-                    [Op.in]: topicsToAssociate.map(({ id }) => id!),
-                  },
+            const createPermissionsPromises = topicsToAssociate.map((topic) => {
```

Comment:
Ok so can we use type assertions or just raw SQL instead?

Reply:
Maybe we can use raw sql? But I need to make a prepared statement for each allowed_actions array. Is there an easy way to do this? Because the current way we use prepared statements is by specifying the individual substitution variables, but this is not possible to do in a loop I think.

Reply:
`bulkCreate` seemed to work for me when I tried it but maybe I did something wrong. Either way, we should use a raw query if `bulkCreate` doesn't work. Not sure what you mean by a prepared statement. If you mean creating a loop to write the query + add the replacements/bindings then yes that is necessary and no there isn't really an easy way that I know of.

Reply:
works with sequelize.literal